docs/01-start-here/01-installation-steps.md

```diff
@@ -116,7 +116,7 @@ You may find it useful to add [this script](https://gist.github.com/sndrs/5940e9
 is present.
 
 Frontend uses a `.nvmrc` file which specifies a given version of node as a requirement. If you use
-[nvm](https://github.com/creationix/nvm#install-script) to manage multiple versions of Node on your machine, you can run `nvm use` to swich to the version specified by `.nvmrc`. If that version number is not available locally, you might have to install the missing version with `nvm install <version number>`.
+[nvm](https://github.com/creationix/nvm#install-script) to manage multiple versions of Node on your machine, you can run `nvm use` to swich to the version specified by `.nvmrc`. If that version number is not available locally, you might have to install the missing version with `nvm install <version number>`. (As a result of upgrades to the new M1 machines you may be using `asdf` or `fnm`).
 
 Alternatively you could use your system Node, for instance using the following:
```

Comment:
I think we can future proof this a bit better by referring the user to the .nvmrc file for the current version?:

```suggestion
Alternatively you could directly install Node on your system.

See [.nvmrc](https://github.com/guardian/frontend/blob/main/.nvmrc) for the current version and install that version using your preferred package manager.

For example:
```


Mac:
```bash
$ brew install node@[current-version]
```
e.g.
```bash
$ brew install node@18.16.0
```

Reply:
Sorry the formatter got a bit confused here with all the nested backticks. Happy to talk you through it

Reply:
Not at all that looks great, thanks @arelra